container-accesslogging/src/main/java/com/yahoo/container/logging/TraceRenderer.java

```diff
@@ -0,0 +1,173 @@
+package com.yahoo.container.logging;
+
+import com.yahoo.yolean.trace.TraceNode;
+import com.yahoo.yolean.trace.TraceVisitor;
+import com.fasterxml.jackson.core.JsonGenerator;
+
+import java.io.IOException;
+
+public class TraceRenderer extends TraceVisitor {
```

Comment:
Consider `TraceRenderer` to yolean (unless it has dependencies to types in `container-accesslogging`). Both `TraceVisitor` and `TraceNode` are classes from yolean.